test-support/page-object/properties/click-on-text.js

```diff
@@ -1,8 +1,28 @@
 import Ember from 'ember';
 import { buildSelector } from '../helpers';
 
+/* global wait, find, click */
+
 var { merge } = Ember;
 
+function findChildElement(tree, selector, textToClick, options) {
+  // Suppose that we have something like `<form><button>Submit</button></form>`
+  // In this case <form> and <button> elements contains "Submit" text, so, we'll
+  // want to __always__ click on the __last__ element that contains the text.
+  var selctorWithSpace = (selector || '') + ' ';
+  var fullSelector = buildSelector(tree, selctorWithSpace, merge({ contains: textToClick, last: true }, options));
```

Comment:
I prefer `var` everywhere, I think it's less confusing. Maybe we should adhere to a code convention in the future.